network/network.go

```diff
@@ -11,6 +11,9 @@ const (
 	CNIImage         = "cniImage"
 	NodeImage        = "nodeImage"
 	ControllersImage = "controllersImage"
+	CalicoctlImage   = "calicoctlImage"
 	FlannelImage     = "flannelImage"
 	FlannelCNIImage  = "flannelCNIImage"
+	CloudProvider    = "cloudprovider"
```

Comment:
I believe CloudProvider constant is defined in cluster/network.go already? is there a reason why we are redefining it here?

Reply:
Yes, the same constant name is used, but they don't necessarily have to have the same value, this is simply to avoid sending constants across packages.